domains/astra.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "MeHiGalatHu"
```

Comment:
You need to have an email or an alternative, but the email field should say. Refer to docs for more info: https://docs.is-a.dev